plugins/mcreator-localization/lang/texts.properties

```
@@ -267,6 +271,7 @@ dialog.image_maker.save_as_new=Save as a new texture
 dialog.image_maker.generate_from_template=Generate from template
 dialog.image_maker.texture_kind=What kind of texture is this?
 dialog.image_maker.texture_type=Texture type
+dialog.image_maker.texture_type.ok=Select this texture type
```

I'd suggest changing translation keys:
```suggestion
dialog.image_maker.texture_type.message=What kind of texture is this?
dialog.image_maker.texture_type.title=Texture type
dialog.image_maker.texture_type.select=Select this texture type
```

---

Is it really needed ? Also, when you change a key, you change it where it's used.

---

All the keys for dialogs I remember use `*.message` for dialog message, `*.title` for dialog title and `*.confirm` or `*.ok/cancel` for dialog buttons.

---

Ok, I will change it later (your commit will cause a build error).

---

Right, because I can't change all the occurrences. I just showed it to you.